contracts/modules/validators/K1Validator.sol

```diff
@@ -226,20 +225,10 @@ contract K1Validator is IValidator, ERC7739Validator {
     /// @param hash The hash of the data to validate
     /// @param signature The signature data
     function _validateSignatureForOwner(address owner, bytes32 hash, bytes calldata signature) internal view returns (bool) {
-        // Check if the 's' value is valid
-        bytes32 s;
-        assembly {
-            // same as `s := mload(add(signature, 0x40))` but for calldata
-            s := calldataload(add(signature.offset, 0x20))
-        }
-        if (uint256(s) > 0x7FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF5D576E7357A4501DDFE92F46681B20A0) {
-            return false;
-        }
-
-        if (SignatureCheckerLib.isValidSignatureNowCalldata(owner, hash, signature)) {
+        if (hash.recover(signature) == owner) {
```

Comment:
@filmakarov does it make sense to put this under try catch. because solady ECDSA lib could revert, and we don't want reverts.

Reply:
makes sense.

Reply:
done